04/2_print_string_buffer/buffered_print.c

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
+#include <string.h>
+
+#include "buffered_print.h"
+
+#define BUFFER_LENGTH (16)
+
+static int s_buffer_chars = 0;

```

Comment:
s_buffer_index

Reply:
this will be pointing to the index where next str*** func should start to write from